src/main/java/com/pinterest/secor/io/impl/OrcProtoReaderWriterFactory.java

```diff
@@ -0,0 +1,147 @@
+package com.pinterest.secor.io.impl;
+
+import java.io.IOException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.hive.ql.exec.vector.VectorizedRowBatch;
+import org.apache.hadoop.io.compress.CompressionCodec;
+import org.apache.orc.OrcFile;
+import org.apache.orc.Reader;
+import org.apache.orc.RecordReader;
+import org.apache.orc.TypeDescription;
+import org.apache.orc.Writer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.common.base.Throwables;
+import com.google.protobuf.Message;
+import com.pinterest.secor.common.LogFilePath;
+import com.pinterest.secor.common.SecorConfig;
+import com.pinterest.secor.io.FileReader;
+import com.pinterest.secor.io.FileReaderWriterFactory;
+import com.pinterest.secor.io.FileWriter;
+import com.pinterest.secor.io.KeyValue;
+import com.pinterest.secor.util.OrcProtoUtil;
+import com.pinterest.secor.util.OrcUtil;
+
+public class OrcProtoReaderWriterFactory implements FileReaderWriterFactory{
+
+	private static final Logger LOG = LoggerFactory.getLogger(OrcUtil.class);
```

Comment:
The indentation seems inconsistent with rest of the classes.  In secor, we use 4 space indentation (not TAB).